src/main/charts/confluence/values.yaml

```diff
@@ -1005,6 +1005,9 @@ synchrony:
         # -- Initial Memory request Synchrony pod
         #
         memory: "2.5G"
+      #  limits:
```

Comment:
@LaszloCsorbaBearingPoint can you please fix indentation here so that when you uncomment limits are on the same level with requests?

Reply:
Sure, I just did that, and I also fixed the same for the Confluence container as well as it was the same (from where I copied over the same syntax)